pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -326,42 +334,46 @@ func (r *ReconcileComplianceScan) phaseLaunchingHandler(h scanTypeHandler, logge
 	logger.Info("Phase: Launching")
 
 	scan := h.getScan()
-	err = createConfigMaps(r, scriptCmForScan(scan), envCmForScan(scan), envCmForPlatformScan(scan), scan)
-	if err != nil {
-		logger.Error(err, "Cannot create the configmaps")
-		return reconcile.Result{}, err
-	}
 
-	if err = r.handleRootCASecret(scan, logger); err != nil {
-		logger.Error(err, "Cannot create CA secret")
-		return reconcile.Result{}, err
-	}
+	// check the scanner type of the scan
+	if scan.Spec.ScannerType == compv1alpha1.ScannerTypeOpenSCAP || scan.Spec.ScannerType == "" {
```

Comment:
Do we ever have a scenario where we set scanner type to "" and make it to this point?

Shouldn't we always have a known scanner?

Reply:
no we won't have "", we will default to have OpenSCAP